core/src/main/scala/org/apache/spark/scheduler/BlacklistTracker.scala

```diff
@@ -168,6 +169,21 @@ private[scheduler] class BlacklistTracker (
       if (newTotal >= MAX_FAILURES_PER_EXEC && !executorIdToBlacklistStatus.contains(exec)) {
         logInfo(s"Blacklisting executor id: $exec because it has $newTotal" +
           s" task failures in successful task sets")
+        conf.get(config.BLACKLIST_ENABLED) match {
```

Comment:
shouldn't this just be using isBlacklistEnabled? and actually I don't think you will even get here if its disabled because it won't create the BlacklistTracker at all  so this shouldn't be needed